packages/project-build/src/shared/graph-utils.test.ts

```diff
@@ -0,0 +1,148 @@
+// graph-utils.test.ts
```

Comment:
why have a filename in comment?

Reply:
Auto generated test, 99% is written with vscode chat gpt